icons/templates/index.html.hbs

```diff
@@ -7,14 +7,6 @@
     <title>Icons Fonts Preview</title>
     <link rel="stylesheet" href="../css/momentum-ui-icons.css">
     <style>
-      /* Override the @font-face from the link above because it points to unpkg.com */
-      @font-face {
-        font-family: "{{fontName}}";
-        src: url("./momentum-ui-icons.woff2") format("woff2"),
-          url("./momentum-ui-icons.woff") format("woff");
-        font-weight: normal;
-        font-style: normal;
-      }
```

Comment:
No longer necessary since `momentum-ui-icons.css` doesn't link directly to unpgk.com anymore.